plugin/pkg/admission/noderestriction/admission.go

```diff
@@ -309,6 +309,12 @@ func (c *nodePlugin) admitPVCStatus(nodeName string, a admission.Attributes) err
 		oldPVC.Status.Conditions = nil
 		newPVC.Status.Conditions = nil
 
+		// TODO(apelisse): We don't have a good mechanism to
```

Comment:
can you add this to the list of things to look at for beta?

Reply:
Yeah, good idea

Reply:
done